common/web/lm-worker/build.sh

```diff
@@ -35,6 +35,7 @@ display_usage ( ) {
 
 WORKER_OUTPUT=build
 WORKER_OUTPUT_FILENAME=$WORKER_OUTPUT/index.js
+WORKER_TEST_BUNDLE_TARGET_FILENAME=$WORKER_OUTPUT/index.ts-nocheck.ts
```

Comment:
I would like a clearer name than `index.ts-nocheck.ts`. That doesn't describe the what module is but instead just the hack that it uses to make it work :grin:

Reply:
I would too, honestly.  Just, I was drawing blanks as to what that name would even be.

Reply:
How about `index.wrapped-for-bundle.ts` (long but points in the right direction, and as the name is internal to the build process, it won't leak into distribution)